plasmapy/particles/data/test/__init__.py

```diff
@@ -3,5 +3,5 @@
 
 import plasmapy
 
-rootdir = os.path.join(os.path.dirname(plasmapy.__file__), "data", "test")
+rootdir = os.path.join(os.path.dirname(plasmapy.__file__), "particles", "data", "test")
 file_list = glob.glob(os.path.join(rootdir, "*.[!p]*"))
```

Comment:
Looking at this now, I bet this would fail with wheels, eggs or whatever... we should probably be using something like `pkgutil.get_data("plasmapy.particles.data.test", filename)` with an explicit list of files to be loaded.

Although... maybe I'm wrong, but it looks like we're not using this `file_list` line at all. Only `rootdir` is used; and that's in `plasmapy/classes/sources/tests/test_openpmd_hdf5.py`, where we test `HDF5Reader`.

Nevermind for now; I'll refactor this later.

Reply:
I made changes changes to how `rootdir` is defined, which I think addresses your issues here and #824.  I also renamed `rootdir` -> `data_dir` to be a little more explicit.